### PR TITLE
Add test for "search index = fromIndex"

### PR DESCRIPTION
**Pull request by gestap**

### CODE REVIEW

File: src/arrayMethodIndexOf.test.js [hunk 1]
```diff
@@ -48,3 +48,8 @@ test(`[].indexOf2() = -1`, () => {
   expect([].indexOf2())
     .toBe(-1);
 });
+
+test(`indexOf2(44, 4) = 4 fromIndex 4`, () => {
```
Comment (roman-mate) [inline src/arrayMethodIndexOf.test.js]: description should be human readable. 
"returns element when it's on a fromIndex position"

let's update for other test case description to be human readable too.

Comment (gestap) [reply src/arrayMethodIndexOf.test.js]: done

File: src/arrayMethodIndexOf.test.js [hunk 2]
```diff
@@ -14,37 +14,43 @@ test(`indexOf2 doesn't call default indexOf`, () => {
...
 });
 
-test(`indexOf2('1') = -1`, () => {
+test(`should return -1 when only substring of the element is present`, () => {
   expect(source.indexOf2('1'))
     .toBe(-1);
 });
 
-test(`indexOf2('11', 1) = -1 fromIndex 1`, () => {
+// eslint-disable-next-line max-len
```
Comment (roman-mate) [inline src/arrayMethodIndexOf.test.js]: You pretty much never want to disable eslint errors, eslint enforces good practices which make the code more readable. 

Just add a line break to make this line shorter.

Comment (gestap) [reply src/arrayMethodIndexOf.test.js]: done

**Pull request merged**